examples/vision/sam.py

```diff
@@ -0,0 +1,601 @@
+"""
+Title: Segment Anything Model with 🤗Transformers
+Author: [Sayak Paul](https://twitter.com/RisingSayak) & [Merve Noyan](https://twitter.com/mervenoyann)
```

Comment:
```suggestion
Authors: [Merve Noyan](https://twitter.com/mervenoyann) & [Sayak Paul](https://twitter.com/RisingSayak)
```

You have put in a lot of effort in this example. You should be the primary author.

Reply:
I have mad respect for this gesture. Who else but @sayakpaul!